docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_exposure/index.rst

```diff
@@ -7,3 +7,172 @@ Iron Deficiency Risk Exposure
+
+Notably, there are two different approaches to the estimation of the iron deficiency risk exposure in GBD 2019. We will refer to these as *Global TMREL exposure estimation* and *location-specific TMREL exposure estimation*. Of note, the *Global TMREL exposure estimation* approach was used for the calculation of the iron deficiency exposure in GBD 2019, although the *location-specific TMREL exposure estimation* approach is described in the GBD 2019 Risk Factor methods appendix. Each approach is described below.
+
+Regardless of the exposure estimation approach, the iron deficiency risk factor is defined as a **continuous distribution of hemoglobin concentrations in g/L**.
+
+.. image:: iron_risk_hierarchy.svg
+
+Global TMREL Exposure Estimation
+++++++++++++++++++++++++++++++++
+
+This estimation approach is not described in the GBD 2019 Risk Factor Methods Appendix, but is documented in the IHME stash repo for the iron deficiency risk factor, `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/iron_deficiency>`_, specifically in the `calculate_normal_hgb.py` and `nutrition_iron_exposure.py` files. For the GBD 2019 estimation of the iron deficiency risk factor exposure, the following was performed:
+
+#. Estimate the TMREL
+
+	The iron deficiency TMREL was estimated as the 95th percentile of mean population hemoglobin concentration among all GBD locations for each year-, sex-, and age-specific demographic group. If the observed mean population hemoglobin concentration for a given location was greater than the 95th percentile, then the observed population hemoglobin concentration was used as the TMREL for that location. See the specific code `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/iron_deficiency/calculate_normal_hgb.py>`_.
+
+	This value is stored as the TMREL for the iron deficiency risk factor in GBD 2019.
+
+#. Calculate iron deficiency risk exposure
+
+	The iron deficiency risk factor is calculated according to the following equation, as documented `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/iron_deficiency/nutrition_iron_exposure.py>`_ and externally validated `here <https://github.com/ihmeuw/sim_sci_maternal_anemia/blob/master/data_validation/maternal_disorders_burden/custom_paf_calculations.ipynb>`_.
+
+	:math:`exposure = TMREL - (p_* * (TMREL - mean_\text{Hgb}))`
```

Comment:
The outer set of parentheses is unnecessary (maybe a bit nitpicky, but it might be a bit more readable without them)

Reply:
Can "TMREL - mean_{Hgb}" be interpreted as the hemoglobin shift corresponding to the "aggregate cause" consisting of all causes of anemia except iron deficiency?  I'm trying to understand how the global TMREL approach relates to the local TMREL approach described below.

Reply:
No, this is really just the difference in observed hemoglobin in a given location compared to the 97.5th percentile of all observed hemoglobin concentrations globally, which is attributable to all causes of low hemoglobin, including iron deficiency, non iron deficiency, pregnancy, and anything else. This is why this approach is a little non-sensical to me.